include/kernels/field_strength_tensor.cuh

```diff
@@ -1,11 +1,13 @@
 #include <gauge_field_order.h>
 #include <index_helper.cuh>
 #include <quda_matrix.h>
+#include <kernel.h>
 
 namespace quda
 {
 
-  template <typename Float_, int nColor_, QudaReconstructType recon_ > struct FmunuArg
+  template <typename Float_, int nColor_, QudaReconstructType recon_ >
+  struct FmunuArg : kernel_param<1>
```

Comment:
I don't think we need the explicit `<1>`, it looks like the default is `true` in `include/kernel_helper.h` so we should only need a `<>`. Am I understanding this correctly?

Reply:
You are understanding it correctly.  Good catch 😄 .  Fixed with 58d1bcaaa.